src/scripts/main.js

```diff
@@ -1 +1,81 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  // eslint-disable-next-line no-shadow
+  const onLeftClick = (event) => {
+    if (event.button === 0) {
+      resolve('First promise was resolved');
+    }
+  };
+
+  document.addEventListener('mousedown', onLeftClick);
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('contextmenu', (event) => {
+    event.preventDefault();
+    resolve('Second promise was resolved');
```

Comment:
The second promise resolves on a right-click as well. If this is not intended, consider revising the logic to differentiate between left and right clicks.